src/accelerate/state.py

```diff
@@ -130,8 +130,16 @@ def __init__(
                 ), "DeepSpeed is not available => install it using `pip3 install deepspeed` or build it from source"
                 self.distributed_type = DistributedType.DEEPSPEED
                 if not torch.distributed.is_initialized():
-                    torch.distributed.init_process_group(backend="nccl", **kwargs)
+                    from .utils import compare_versions
```

Comment:
This can be imported at the top.

Reply:
that was leading to circular depency error

Reply:
I think it becomes a circular dep import which is why it's down there

Reply:
Ah yes, I can see that! Good for me then!